package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "init": "git submodule update --init --recursive",
+		"init": "git submodule update --init --recursive && pnpm install && pnpm run build --workspace packges/bos-workspace",
```

Comment:
Did this work? There is typo in packges -- please confirm it works

Reply:
Fixed it